modiscript/api.py

```diff
@@ -0,0 +1,23 @@
+from .lexer import Lexer
+from .parser import Parser
+
+
+class ModiScript:
+    def __init__(self, debug=False):
+        self.debug = debug
+
+    def _compile_file(self, value, valueType="filename"):
```

Comment:
A small change but what fo you think about using snake_case for `valueType`. Because Modiji **MIGHT** hate camel.